02-overview-surveys.Rmd

```diff
@@ -149,33 +151,33 @@ To avoid this issue, researchers should consider these possibilities and adjust
 
 ## Post Survey Processing {#overview-post}
 
 After data collection, a variety of activities need to be conducted before we can analyze the survey. Multiple decisions made during this post-survey phase can assist researchers in reducing different error sources, such as through weighting to account for the sample selection. Knowing the decisions researchers made in creating the final analytic data can impact how analysts use the data and interpret the results.
 
 ### Data Cleaning and Imputation {#overview-post-cleaning}
 
-Post-survey cleaning and imputation is one of the first steps researchers will do to get the survey responses into a dataset for use by analysts. Data cleaning can comprise cleaning inconsistent data (e.g., with skip pattern errors or multiple questions throughout the survey being consistent with each other), editing numeric entries or open-ended responses for grammar and consistency, or recoding open-ended questions into categories for analysis. Each project should create their own rules for handling different cleaning situations, and no specific rules must be followed. Instead, researchers should use their best judgment to ensure data integrity remains, and all decisions should be documented and available to those using the data in the analysis. Each decision a researcher makes impacts *processing error*, so often researchers will have multiple people review these rules or recode open-ended data and adjudicate any differences in an attempt to reduce this error.
+Post-survey cleaning and imputation is one of the first steps researchers will do to get the survey responses into a dataset for use by analysts. Data cleaning can comprise of cleaning inconsistent data (e.g., with skip pattern errors or multiple questions throughout the survey being consistent with each other), editing numeric entries or open-ended responses for grammar and consistency, or recoding open-ended questions into categories for analysis. Each project should create their own rules for handling different cleaning situations, and no specific rules must be followed. Instead, researchers should use their best judgment to ensure data integrity remains, and all decisions should be documented and available to those using the data in the analysis. Each decision a researcher makes impacts *processing error*, so often researchers will have multiple people review these rules or recode open-ended data and adjudicate any differences in an attempt to reduce this error. 
```

Comment:
Each project... sentence and the following one I'm not following.  Need to review this and check edits to see if something was accidentally removed/edited along the way.